influxdb/line_protocol.py

```diff
@@ -40,8 +42,7 @@ def _convert_timestamp(timestamp, precision=None):
             return ns / 1e9 / 60
         elif precision == 'h':
             return ns / 1e9 / 3600
-
-    raise ValueError(timestamp)
+        raise ValueError(timestamp)
```

Comment:
This should be outdented, otherwise what happens if we don't hit any of the first two `isinstance` checks?